MODULE.bazel

```diff
@@ -1,22 +1,22 @@
-module(name = "com_github_google_benchmark", version="1.8.2")
+module(name = "google_benchmark", version="1.8.2")
 
 bazel_dep(name = "bazel_skylib", version = "1.4.1")
 bazel_dep(name = "platforms", version = "0.0.6")
 bazel_dep(name = "rules_foreign_cc", version = "0.9.0")
 bazel_dep(name = "rules_cc", version = "0.0.6")
-bazel_dep(name = "rules_python", version = "0.24.0")
-bazel_dep(name = "googletest", version = "1.12.1", repo_name = "com_google_googletest")
+bazel_dep(name = "rules_python", version = "0.24.0", dev_dependency = True)
```

Comment:
Yeah, I am torn on this as well. The question is how people will use the Python library in their projects (iff they rely on Bazel).

Since there is a PyPI package for this module, my assumption was this would be the preferred way of depending on the library, instead of pulling in the github source code and depending on the BUILD targets directly. But that is my personal speculation. 

If you think the preferred way should be the second option, I am happy to remove it here. 
WDYT?

Reply:
i think you're right. leave it as a dev dependency. thanks.